src/ql/arch/cc/resources/hwconf_default.json

```diff
@@ -829,42 +829,7 @@
         }
     },
     "topology": {
-        "edges": [
-            { "id":  0, "src":  2, "dst":  0 },
-            { "id":  1, "src":  2, "dst":  1 },
-            { "id":  2, "src":  2, "dst":  3 },
-            { "id":  3, "src":  2, "dst":  4 }
-        ]
     },
```

Comment:
It should be legal to just remove the whole topology key nowadays. Same for resources.